### ISSUE TITLE
`about` throws exception for values of type `Char`.

### ISSUE THREAD

**Issue by loonatick-src** [tecosaur/About.jl#30]
```
julia> about('C')
Char (<: AbstractChar <: Any), occupies 4B.
ERROR: bitcast: target type not a leaf primitive type
Stacktrace:
 [1] reinterpret
   @ .\essentials.jl:513 [inlined]
 [2] memorylayout(io::Base.TTY, char::Char)
   @ About C:\Users\Chaitanya Kumar\.julia\packages\About\52iMk\src\values.jl:396
 [3] about(io::Base.TTY, value::Char)
   @ About C:\Users\Chaitanya Kumar\.julia\packages\About\52iMk\src\values.jl:43
 [4] about(x::Char)
   @ About C:\Users\Chaitanya Kumar\.julia\packages\About\52iMk\src\About.jl:201
 [5] top-level scope
   @ REPL[37]:1
```
**Version info:**
```
(@v1.9) pkg> st
Status `C:\Users\Chaitanya Kumar\.julia\environments\v1.9\Project.toml`
  [69d22d85] About v1.0.0
```

**Comment by tecosaur** [comment]
Ah yes, you can't `reinterpret(NTuple{...}, ...` in 1.9. Hmm, now that 1.10 is the LTS I'll probably just drop 1.9 compat.

**Comment by tecosaur** [comment]
Eh, suppose I might as well. Fixed as of 8196a75bf1b864e778ac4c1a4ad5e5770e0ee1d6.
